### DISCUSSION TITLE
Is there any tutorial available to explain the structure and how to modify it

### DISCUSSION THREAD

**Discussion by sayedsadat344** [coolsam726/jetstream-inertia-generator#46] [General]
I saw the GitHub repository of this template which looks quite good, but I don't know how to modify it, since there is no proper documentation available.

**Comment by coolsam726** [comment]
@sayedsadat344 Thanks for starting the discussion. Documentation is currently a WIP and will probably be published before end of January. In the meantime I will try to explain in details the structure and how the generator works in this discussion.

**Reply by sayedsadat344** [reply]
Thanks, we will wait for the complete documentation.  Meanwhile, a blog post will help a lot to describe the complete structure, there are confusion in the front & backend redirections in the dashboard, confusion in the components of jetstream & jig components, both directories having the same component, one can not understand where to modify.
